molecule/testinfra/app/test_app_network.py

```diff
@@ -14,8 +14,11 @@
 @pytest.mark.skip_in_prod
 def test_app_iptables_rules(host):
 
+    local = host.get_host("local://")
+
     # Build a dict of variables to pass to jinja for iptables comparison
     kwargs = dict(
+        ssh_ip=local.interface("eth0").addresses[0],
```

Comment:
As above, it's not clear to me why this is suddenly required for Qubes, but not for other staging envs.

Reply:
This change [appears to be breaking CI](https://github.com/freedomofpress/securedrop/pull/5712#pullrequestreview-587293767), investigating.